### PR TITLE
MGMT-1655 - add additional network interface to master and worker VMs

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: discovery-infra/delete_nodes.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def delete_nodes(tfvars):
             virsh_cleanup.DEFAULT_SKIP_LIST,
             [
                 tfvars.get("cluster_name", consts.TEST_INFRA),
-                tfvars.get("libvirt_network_name", consts.TEST_INFRA),
+                tfvars.get("libvirt_network_name", consts.TEST_INFRA_NETWORK),
+                tfvars.get("libvirt_network_name_2", consts.TEST_INFRA_NETWORK_2),
```
Comment (tsorya) [inline discovery-infra/delete_nodes.py:47]: not sure _2 is a good name, can't we have something more descriptive ? :)

**Comment by tsorya** [comment]
@RazRegev please go through.

**Comment by eranco74** [comment]
Why?
Our installation doesn't require a second interface.

File: discovery-infra/delete_nodes.py (hunk 1 above)
Comment (yevgeny-shnaidman) [reply discovery-infra/delete_nodes.py:47]: i am opened to suggestions

**Comment by yevgeny-shnaidman** [comment]
@eranco74 2 reasons:
1) 99.99 % of HW server have 2 and more network interfaces, and we want to be a close as possible to that scenario. for example,if we had 2 interface we could have debugged ETCD issue on VMs and not real Baremetal environment
2) until BMH team finish with their side of updating BMO to run without provisioning interface ( which will probably be pushed to 4.7), we need additional interface to be able to reach "green" cluster installation

File: discovery-infra/delete_nodes.py (hunk 1 above)
Comment (YuviGold) [reply discovery-infra/delete_nodes.py:47]: I don't like it either.. Can it be an array?

Comment (yevgeny-shnaidman) [reply discovery-infra/delete_nodes.py:47]: @YuviGold what do you mean by array? We will still need explicit names for each one

Comment (tsorya) [reply discovery-infra/delete_nodes.py:47]: it can be array in terraform but it will be much more complicated to implement i think

Comment (YuviGold) [reply discovery-infra/delete_nodes.py:47]: > @YuviGold what do you mean by array? We will still need explicit names for each one

You said you want to support multiple networks

Comment (yevgeny-shnaidman) [reply discovery-infra/delete_nodes.py:47]: after discussion with @ronniel1  i am cancelling this pull request, and  the second interface will be added by @RazRegev as part of his task

**Comment by yevgeny-shnaidman** [comment]
after discussion with @ronniel1 i am cancelling this pull request, and the second interface will be added by @RazRegev as part of his task

**Pull request closed without merging**